qiskit/visualization/utils.py

```diff
@@ -193,8 +141,27 @@ def _get_layered_instructions(circuit, reverse_bits=False, justify=None, idle_wi
     return qregs, cregs, ops
 
 
+def _sorted_nodes(dag_layer):
+    """Convert DAG layer into list of nodes sorted by node_id
+    qiskit-terra #2802
+    """
+    dag_instructions = dag_layer['graph'].op_nodes()
+    # sort into the order they were input
+    dag_instructions.sort(key=lambda nd: nd._node_id)
+    return dag_instructions
+
+
+def _is_multibit_gate(node):
```

Comment:
I can't see this method being used anywhere, can it be removed?

Reply:
removed